.travis.yml

```diff
@@ -20,14 +18,25 @@ after_success:
   --podspec MiniApp.podspec
   --readme README.md
 
+deploy:
+  provider: script
+  skip_cleanup: true
+  script: |
+    pod spec lint --allow-warnings 
+    && pod trunk push --allow-warnings --verbose
+  on:
+    tags: true
+    condition: $TRAVIS_TAG =~ ^v
```

Comment:
https://guides.cocoapods.org/making/making-a-cocoapod.html#library-versioning suggests that prefixing x.y.z numbers with `v` isn't standard and I'm not sure it's supported by CocoaPods dependency resolver. Can you confirm?

Reply:
I've also updated the `source` to prepend `v` to the tag. I found this solution here: https://stackoverflow.com/a/22646192
I also tested this by adding a tag on my fork and updating the podspec to point to my fork. After doing this, `pod spec lint` passed. Does this mean it should work?

Reply:
Also, I saw that Cocoapod's examples use `v` in the tag: https://guides.cocoapods.org/syntax/podspec.html